### ISSUE TITLE
Vocab addition for Claims Type Concepts (T1 and T7 combined)

### ISSUE THREAD

**Issue by dangelelli** [OHDSI/Vocabulary-v5.0#1081]
_The issue should briefly describe proposed changes or additions, rationale behind them, and potential impact on the activities in OHDSI (such as research and ETL). This will help to identify other contributors that may be working on the same content and/or provide suggestions and recommendations at the initial stage of development._

# Description + Rationale of Proposed Change
## My goal
Annotate records with provenance showing that they are from a particular claim type and include the “open”/“closed approved” / "closed rejected" adjudication status. As an example, a specific record may come from a “approved professional claim line” or an “open dental claim line”.
## The issue
This is not currently possible in the OMOP schema using standard vocabularies. The standard vocabulary Type Concept provides the ability to delineate claim type (id: `32871` “Professional claim”, id: `32844` “Facility claim”, id: `32816` “Dental claim”, etc.). That same vocabulary also has records for id: `32875` “Payer System” and id: `32867` “Primary Provider System” which can roughly translate to “open” pre-adjudicated claims or “closed” adjudicated claims. The trouble is that there aren’t any vocabulary entities that combine this information.
See accepted type concepts for reference: [here](https://athena.ohdsi.org/search-terms/terms?domain=Type+Concept&standardConcept=Standard&page=1&pageSize=15&query=)
## Why is it important to retain this information?
TLDR - in a data model combining multiple sources, it is critical to enable researchers to select down to appropriate populations. Two main vectors for that selection that cut across many OMOP tables and closely reflect provenance are below:

### Claim type (Professional, Facility, Dental, etc.)
Provide researchers with a general understanding of provenance
Understanding claim type is critical for researching things like policy interventions, utilization patterns, etc.
### Adjudication status
Post-adjudication (“Closed”) claims are generally viewed as higher quality for most use cases. They represent completed transactions that have been thoroughly reviewed, with any discrepancies resolved. They can be either “Approved” or “Rejected”.
Pre-adjudication (“Open”) claims frequently have more data, but also missing claims and duplicates which affect research findings if not properly understood and managed.

Also, adjudication status is critical for the cost table to work correctly.

### Spreadsheet Templates
- [t1_adding_concept-claim-types](https://docs.google.com/spreadsheets/d/1n_ftOjG93pNjOGmOYkupvDSD4mNWTwXUVSelyow55Jo/edit?usp=sharing)
- [t7_promoting_nonstadard_to_standard-claim-types](https://docs.google.com/spreadsheets/d/1hmRGtef7IiCIkFAEZgnDJb0kJeNE9f5-ACL4qqCsZMQ/edit?usp=sharing) (builds on the T1)

### Description of new members
I am requesting that the standard vocabulary for Type Concepts be expanded to include entities that can create a hierarchical relationship grouping by claim adjudication status.
I understand that this is a two-step process. I have populated a T1 template for adding the non-standard concepts. I’ve also populated a T7 template for promoting those same concepts to Standard.
An example of the new members I am requesting is below, with new entities labeled with a *. (This listing is entirely supplemental to the T1 and T7 attachments and is only shown for explanatory purposes)

- Dental claim
  - Approved dental claim*
  - Rejected dental claim*
  - Pre-adjudicated dental claim*
- Facility claim
  - Approved facility claim*
  - Rejected facility claim*
  - Pre-adjudicated facility claim*
- Inpatient claim
  - Approved inpatient claim*
  - Rejected inpatient claim*
  - Pre-adjudicated inpatient claim*
- Outpatient claim
  - Approved outpatient claim*
  - Rejected outpatient claim*
  - Pre-adjudicated outpatient claim*
- Pharmacy claim
  - Approved pharmacy claim*
  - Rejected pharmacy claim*
  - Pre-adjudicated pharmacy claim*
- Professional claim
  - Approved professional claim*
  - Rejected professional claim*
  - Pre-adjudicated professional claim*
- Vision claim
  - Approved vision claim*
  - Rejected vision claim*
  - Pre-adjudicated vision claim*

The leaf-level new members shown above would also roll up to parents representing adjudication status in a separate hierarchy, shown below:

- Adjudicated Approved Claims*
  - Approved dental claim *
  - Approved facility claim *
  - Approved inpatient claim *
  - Approved outpatient claim *
  - Approved pharmacy claim *
  - Approved professional claim *
  - Approved vision claim *
- Adjudicated Rejected Claims*
  - Rejected dental claim
  - Rejected facility claim *
  - Rejected inpatient claim *
  - Rejected outpatient claim *
  - Rejected pharmacy claim *
  - Rejected professional claim *
  - Rejected vision claim *
- Pre-Adjudicated Claims*
  - Pre-Adjudicated dental claim *
  - Pre-Adjudicated facility claim *
  - Pre-Adjudicated inpatient claim *
  - Pre-Adjudicated outpatient claim *
  - Pre-Adjudicated professional claim *
  - Pre-Adjudicated pharmacy claim *
  - Pre-Adjudicated vision claim *

## Relevant references
Existing standard “Type Concept” list: [here](https://athena.ohdsi.org/search-terms/terms?domain=Type+Concept&standardConcept=Standard&page=1&pageSize=15&query=)
An OMOP forum post discussing this proposal: [here](https://forums.ohdsi.org/t/vocab-request-additional-type-concepts-for-claims-provenance/22584)
# Potential impact on the activities in OHDSI
## If T1 is accepted
No concepts are deprecated, minimal impact on OHDSI activities
Would allow complex claims-specific use cases to more accurately describe data, although they would be using non-standard concepts to do so
## If T7 is accepted
No concepts are deprecated, minimal impact on OHDSI activities
Would allow complex claims-specific use cases to more accurately describe data
May require manual review if additional “claim” type concepts are added in the future. This would be a part of the existing community contributions to vocabularies process.

**Comment by aostropolets** [comment]
Hello-hello! Thanks for contributing! As Christian and Melanie pointed out on forums we as a community should think about the way we want to model this information be it through combination of fields, concepts or else. As far as I see the consensus has not been reached yet so I'd suggest creating a proposal, collecting opinions and coming to Themis/CDM WG/Vocab WG to ratify it first.

**Comment by aostropolets** [comment]
I think you'd go with continuing the conversation you started on the forums to reach a consensus (eg does this info belong to COST table or not). You can also carry out the conversation by coming to the CDM WG or Healthcare Systems WG (https://www.ohdsi.org/upcoming-working-group-calls/). You also need a use case from the researchers who will use the data.

Two reasons: a) you can contribute non-standard concepts but promoting them to standard requires community ratification and b) creating permutations of all claim types sounds like a bad practice from normalization perspective.

**Comment by m-khitrun** [comment]
Hello @dangelelli! Have you made any progress on your issue?

**Comment by dangelelli** [comment]
yes! After additional conversation on the forum, I've got a path forward that uses an extension column for claims status. Given that, I can withdraw this request.
